geoext.php

```diff
@@ -66,6 +67,49 @@ function GeoLocateAP($bssid, $svcs = null)
 	}
 	return $coords;
 }
+function GetFromYandexLocator($bssid) {
+	geoDbg("yandex_locator: $bssid");
+	$url = 'http://api.lbs.yandex.net/geolocation';
+	$apiKey = ''; //API-ключ Yandex Locator
```

Comment:
API keys and configuration should be in this file: https://github.com/binarymaster/3WiFi/blob/master/config.php-distr

You can add it after `YMAPS_APIKEY` there.